### ISSUE TITLE
Validation for classic (non-extended) crontab 

### ISSUE THREAD

**Issue by bflorat** [jmrozanec/cron-utils#161]
Hi,
cron-util 5.0.5

I'd like to validate crontab (UNIX)  expressions in classic mode so I can check expressions before adding them to crontab command but cron-utils seems to use the UNIX extended mode. In classic  mode, among others things,  ? is not supported.

Ex : * * * * ? should fail but pass

Code  :  
```
CronDefinition cronDefinition = CronDefinitionBuilder.instanceDefinitionFor(CronType.UNIX);
CronParser parser = new CronParser(cronDefinition);
Cron quartzCron = parser.parse("* * * * ?");
quartzCron.validate();
```

-> no errros

in crontab -e (RHEL 6) : 

```
* * * * ? /bin/bash
```
Result : 
```
crontab: installing new crontab
"/tmp/crontab.VniXIS":9: bad day-of-week
errors in crontab file, can't install.
Do you want to retry the same edit? 
```

Thanks

**Comment by jmrozanec** [comment]
@bflorat thank you for reporting this. Will try to reproduce the case asap and get back. Thanks!

**Comment by jmrozanec** [comment]
@bflorat we just solved the issue. Thank you for pointing out. Will be released as soon as we close #157 Thanks!

**Comment by bflorat** [comment]
Thanks for your reactivity. I confirm this is fixed in 5.0.6-SNAPSHOT.
